app/views/users/show.html.erb

```diff
@@ -1,39 +1,36 @@
-<main>
-  <section>
-    <div class="card">
-      <div class="user-image border">
-        <%= image_tag(@user.photo, class: "add-css-class") %>
-      </div>
-      <div class="user-data border">
-        <h1><%= @user.name %></h1>
-        <p class="number-posts">Number of posts: <%= @user.post_counter %></p>
-        <div class="right">
-          <%= link_to "Add a post", new_user_post_path(@user), class: "btn right" %>
-        </div>
-      </div>
+<div class="bg-dark min-vh-100 text-white" >
+  <div class="container px-2 py-5">
+  
+  <%= render 'user' %>
+
+  <h1><%=@current_user.name%>  </h1>
```

Comment:
- [ ] Kindly go through your to avoid unnecessary errors. This should be `current_user` without the '@'.

  ![image](https://github.com/Marcraphael12/rails-blog-app/assets/36542986/e5c79580-c490-4cb8-96bc-8e1347a65810)